### ISSUE TITLE
It seems that augmented data has been used in the testing dataset

### ISSUE THREAD

**Issue by hwang9u** [AnshikaAnand222/Speech-Emotion-Recognition#1]
Hello! 🐶 I'm also interested in speech emotion recognition and came across your repository while searching for related resources. I noticed that your code shows high test accuracy, but it seems that augmented data is included in the testing dataset. Shouldn't the augmented data be used only for training and not be included in the testing dataset?

**Comment by AnshikaAnand222** [comment]
Yes, you are correct that ideally, augmented data should only be used for training and not be included in the testing dataset.
You see, this was my college minor project so to improve accuracy I used data augmentation and; trained  and tested on same.
However this is not ideal as this may lead to overfitting and falsely high test accuracy.

**Comment by hwang9u** [comment]
Thank you for your response.

I also worked on the same dataset for a project (it's on my GitHub), and I noticed that although your model is similar to the one I created in the initial stages of the project, the performance difference is significant. I wanted to confirm if my assumptions about the reasons behind this difference are correct. 

I am also curious about the results of attempting augmentation on the training data after dividing the training and test data in advance. Thank you again for your response. 😀
